pkg/gofr/datasource/constants.go

```diff
@@ -0,0 +1,8 @@
+package datasource
+
+// Cassandra batch types.
+const (
+	CassandraLoggedBatch = iota
+	CassandraUnloggedBatch
+	CassandraCounterBatch
+)
```

Comment:
Rather than creating a separate file, we can put it in cassandra.go file, where it is being used.
Moreover, these can be used in cassandra.go file where we have created similar constants for same purpose.

Reply:
I agree